rules/macos/persistence_creation_hidden_login_item_osascript.toml

```diff
@@ -0,0 +1,62 @@
+[metadata]
+creation_date = "2020/01/05"
+maturity = "production"
+updated_date = "2020/01/05"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of osascript to create a hidden login item. This may indicate an attempt to persist a malicious
+program while concealing it's presence.
```

Comment:
```suggestion
Identifies the execution of osascript to create a hidden login item. This may indicate an attempt to persist a malicious
program while concealing its presence.
```